js-exercises/update-object/updateObject.js

```
@@ -0,0 +1,10 @@
+
+function updateObject(replaceIndex, replaceValue, inputArray) {
+  if (replaceIndex > inputArray.length || replaceIndex < inputArray.length * -1) return inputArray;
```

Hey, so what I am suggesting here is completely a developer's choice based on the use case but however line 3 could have thrown an error `(eg: index out of range)` instead of silently returning the same array.
Apart from that could have covered edge cases for typeof `replaceIndex` and `inputArray`. Other than that, this code rocks :guitar: